src/components/VaccineProgressBar/VaccineProgressBar.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+
+export const ProgressBarContainer = styled.div`
+  width: 100%;
+  max-width: 350px;
+  margin: auto;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    max-width: 288px;
+    margin: 0;
+  }
+`;
+
+export const StyledSvg = styled.svg`
+  border: 1px solid ${COLOR_MAP.GREY_2};
+  border-radius: 3px;
```

Comment:
adding `width:100%` to this style block seems to resolve flicker thing (ie seeing a piece of the gray border render/flicker before the bar renders)

Reply:
Oooh, genius!